src/graphics/render_window.rs

```diff
@@ -167,6 +268,24 @@ impl RenderWindow {
     /// All other functions such as `poll_event` or display
     /// will still work (i.e. you don't have to test `is_open`
     /// every time), and will have no effect on closed windows.
+    ///
+    /// # Usage Example
+    ///
+    /// ```no_run
+    /// # use sfml::window::{Event, Style};
+    /// # use sfml::graphics::RenderWindow;
+    /// # // Create a new window
+    /// # let mut window = RenderWindow::new((800, 600),
+    /// #                              "SFML window",
+    /// #                              Style::CLOSE,
+    /// #                              &Default::default());
+    /// while window.is_open() {
+    ///     // Do something
+    ///     window.close();
```

Comment:
I think this is kind of a silly example, because you close it right away on the first iteration.
I think you should close on `Closed` event, even if it makes the example longer.

Reply:
Done